.github/workflows/debian.yml

```diff
@@ -0,0 +1,42 @@
+name: debian
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+      - workflows/debian  # For debugging.
```

Comment:
Ah, yes. The above was for debugging the _workflow_ itself, but you're right it would be useful to have a place to push where the test will be run.

How about `force_ci/debian/**` and `force_ci/all/**`? (And of course `force_ci/build/**` and `force_ci/all/**` for build.yml.)

Reply:
Sounds great to me!